app/src/renderer/vuex/modules/governance/deposits.js

```diff
@@ -27,7 +33,6 @@ export default ({ node }) => {
         Raven.captureException(error)
         state.error = error
       }
```

Comment:
Just asking: In case of errors the state will remain { ..., loading: true}? Shouldn't we keep the state.loading = false in a finally block?

Reply:
Got it! Alternatively I was thinking of a single property ~ enum with the possible states, but probably won't be easy to manage without types.

Reply:
Also possible. Enum:
```
const Enum = {
  Loading: 1,
  Loaded: 2,
  Failed: 3
}

let state = Enum.Loading
```

But we could have an error and still retry loading in the background, so using multiple states at the same time. But I see you will have the better solution here in the end. ;)